openmicroscopy/openmicroscopy
# Original metadata loading

This fixes timeouts when loading original metadata (see https://trac.openmicroscopy.org.uk/ome/ticket/12625). It also moves the metadata loading to "on demand" so that original metadata is not loaded unless needed.

To test:
 - Open "Acquisition" tab for an image (also test well / image) and expand the "Original Metadata" panel.
 - Original metadata should be loaded when panel is expanded, and search filters should work as before.
 - When an omero.LockTimeout is thrown, a 'try again' message is shown and clicking this should load as before (see screenshot). NB: don't know of a good way to test this. I just added ``` raise omero.LockTimeout()``` to the Blitz Gateway image.loadOriginalMetadata.

![screen shot 2015-01-05 at 15 02 39](https://cloud.githubusercontent.com/assets/900055/5614792/8e5f8156-94ec-11e4-9309-7cdfb71570cc.png)
